src/store/modules/victim.ts

```diff
@@ -1,11 +1,26 @@
-import { VuexModule, Module, mutation } from 'vuex-class-component'
-
-@Module({ namespacedPath: 'victim/' })
-export default class VictimModule extends VuexModule {
-  camera = 'camera1'
-
-  @mutation
-  setCamera(cameraName: string) {
-    this.camera = cameraName
-  }
-}
+import { VuexModule, Module, mutation } from 'vuex-class-component'
+
+@Module({ namespacedPath: 'victim/' })
+export default class VictimModule extends VuexModule {
+  cameraMotion = 'camera1'

```

Comment:
Note to self: je commence de moins en moin a aimer la facon que je retrieve des cameras. Va falloir trouver une meilleure facon apres la compé, mais pour l'instant sa fait la job